apps/marketplace/components/BuyerDashboard/BuyerDashboardHeader.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { rootPath } from 'marketplace/routes'
+import styles from './BuyerDashboard.scss'
+
+export class BuyerDashboardHeader extends Component {
```

Comment:
you could make this SFC - no need to connect it - pass the function and the org as props